lib/origen_testers/igxl_based_tester/base.rb

```diff
@@ -717,7 +717,11 @@ def pattern_footer(options = {})
         $tester.align_to_last
         # cycle(:microcode => "#{$dut.end_of_pattern_label}:") if $dut.end_of_pattern_label
         if options[:end_in_ka]
+          if $tester.j750?
```

Comment:
IMO, The initial implementation assumes that the microcode between J750 and UltraFLEX is the same. the microcode keep_alive is specific for J750, Ultraflex has the microcode keepalilve. So, would we have a pattern_footer method in ultraflex.rb and override this there? Just trying to understand here.

same for below, UF does not have a hi lo limit valid argument.